bindings/nodejs/lib/types/wallet/account.ts

```diff
@@ -1,12 +1,13 @@
+// Copyright 2021-2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
 import type { AccountAddress, AddressWithUnspentOutputs } from './address';
 import type { OutputData } from './output';
 import type { Transaction } from './transaction';
-import type {
-    HexEncodedAmount,
-    IOutputResponse,
-    ITransactionPayload,
-} from '@iota/types';
 import { CoinType } from '../../client';
+import { TransactionPayload } from '../block/payload/transaction';
+import { HexEncodedAmount, HexEncodedString } from '../utils';
+import { OutputResponse } from '../models/api';
```

Comment:
'TransactionPayload' is defined but never used
'OutputResponse' is defined but never used